templates/email_report_violation.html

```diff
@@ -0,0 +1,24 @@
+<p>Hello Admin,</p>
```

Comment:
can you include an image of this email being received in the PR description please? Since this is a UI we can see

Reply:
Done